### PR TITLE
TFW-2524 - add 16px discovery icon

### PR DESCRIPTION
**Pull request by tVeratti**

### CODE REVIEW

File: components/icons/discovery.js [hunk 1]
```diff
@@ -1,15 +1,26 @@
...
+          : 'M11 2v10a1 1 0 0 0 1 1h10c0 6.075-4.925 11-11 11S0 19.075 0 13 4.925 2 11 2zm13 9H14a1 1 0 0 1-1-1V0c6.075 0 11 4.925 11 11zM2 13a9 9 0 0 0 17.777 2H12a3 3 0 0 1-3-3V4.223C4.992 5.133 2 8.717 2 13zm19.777-4A9.013 9.013 0 0 0 15 2.223V9h6.777z';
+        break;
+    case 'medium':
+    default: path = filled
+      ? 'M7,2 L7,8 C7,8.55228475 7.44771525,9 8,9 L14,9 C14,12.8659932 10.8659932,16 7,16 C3.13400675,16 0,12.8659932 0,9 C0,5.13400675 3.13400675,2 7,2 Z M9,0 C12.8659932,0 16,3.13400675 16,7 L10,7 C9.44771525,7 9,6.55228475 9,6 L9,0 Z'
+      : 'M6,3.08295844 C3.16228666,3.55903653 1,6.02700371 1,9 C1,12.3137085 3.6862915,15 7,15 C9.97299629,15 12.4409635,12.8377133 12.9170416,10 L8,10 C6.8954305,10 6,9.1045695 6,8 L6,3.08295844 Z M7,2 L7,8 C7,8.55228475 7.44771525,9 8,9 L14,9 C14,12.8659932 10.8659932,16 7,16 C3.13400675,16 0,12.8659932 0,9 C0,5.13400675 3.13400675,2 7,2 Z M10,1.08295844 L10,6 L14.9170416,6 C14.4955231,3.48749513 12.5125049,1.50447689 10,1.08295844 Z M9,0 C12.8659932,0 16,3.13400675 16,7 L10,7 C9.44771525,7 9,6.55228475 9,6 L9,0 Z';
```
Comment (troyblank) [inline components/icons/discovery.js]: This doesn't seem to be compressed, did you run it through something like https://jakearchibald.github.io/svgomg/?

Comment (troyblank) [reply components/icons/discovery.js]: Yea that would help a lot, thanks!

Comment (troyblank) [reply components/icons/discovery.js]: Also did you check for pixel issues? :cringes:

Comment (troyblank) [reply components/icons/discovery.js]: They are seem to be great pixel wise minus the compression.

screen shot 2019-01-28 at 12 35 42 pm

Comment (tVeratti) [reply components/icons/discovery.js]: Updated to compressed versions :)

Comment (troyblank) [reply components/icons/discovery.js]: Awesome, 39% savings is not bad on those character strings :D

**Pull request merged**